addons/mail/static/src/components/follower_subtype/follower_subtype.scss

```diff
@@ -1,27 +1,7 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
-.o_FollowerSubtype_checkbox {
-    margin-inline-end: map-get($spacers, 2);
-}
-
-.o_FollowerSubtype_label {
-    display: flex;
-    flex: 1;
-    flex-direction: row;
-    align-items: center;
-    margin-bottom: map-get($spacers, 0);
-    padding: map-get($spacers, 2);
-}
-
-// ------------------------------------------------------------------
-// Style
-// ------------------------------------------------------------------
-
 .o_FollowerSubtype_label {
-    cursor: pointer;
+    flex-basis: 0;
```

Comment:
Same remark as in `Follower` PR:
https://github.com/odoo/odoo/pull/88196#discussion_r846099802

I don't really get the reasoning of adding this rule.

Reply:
Same answer 🙂  : https://github.com/odoo/odoo/pull/88196#discussion_r846124342

I removed it 👍